app/services/applications/accept.rb

```diff
@@ -71,7 +71,10 @@ def accept_application!
     def reject_other_applications_in_same_cohort!
       return if other_applications_in_same_cohort.blank?
 
-      other_applications_in_same_cohort.update!(lead_provider_approval_status: "rejected")
+      other_applications_in_same_cohort.update!(
+        lead_provider_approval_status: "rejected",
+        reason_for_rejection: Application.reason_for_rejections[:application_accepted_by_other_provider],
```

Comment:
is this the correct reason here?
the `other_applications_in_same_cohort` could be for the same provider.